File: union of two array.cpp
#include<iostream>
#include<vector>
using namespace std;





int main(){

int arr[] = {1,2,3,4,5,6};
int size_Aa = 6;

int arr2[] = {7,8,9,0};
int size_arr2 = 4;

vector<int>ar;

// inserting frist array into vector and after that we will insert 2nd into 
for (int i = 0; i < size_Aa; i++)
{
    /* code */
    ar.push_back(arr[i]);
}

for (int i = 0; i < size_arr2; i++)
{
    /* code */
    ar.push_back(arr2[i]);
}

// now our union is ready and we will print element 
for (int i = 0; i < ar.size(); i++)
{
    /* code */
    cout<<ar[i]<<" ";
}



}

File: union of two array.py


def union(arr1,arr2):
    uni = []
    for i in arr1:
        uni.append(i)
    for i in arr2:
        if i in uni:
            continue
        else:
            uni.append(i)
    return uni


arr1 = [1,2,3,4]
arr2 = [2,3,4,5]
print(union(arr1,arr2))


